topbar.js

```diff
@@ -622,8 +625,15 @@ function setWorkspaceName (name) {
 }
 
 function setMonitor(monitor) {
-    if (!prefs.topbar_follow_focus)
```

Comment:
When `topbar_follow_focus` is true we need to move the top bar to `monitor`. 

 `panelMonitor` also needs to be set as there's other code which depend on knowing which monitor the top bar is on (not necessary each time, but it doesn't hurt).